modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,282 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configure-rollout-route-traffic-using-openshift-routes_{context}"]
+= Configuring Argo Rollouts to route traffic by using OpenShift Routes
+
+You can use OpenShift Routes to configure Argo Rollouts to create routes, rollouts, and services.
+
+The following example procedure creates a route, a rollout, and two services. It then gradually routes an increasing percentage of traffic to a canary version of the application before that canary state is marked as successful and becomes the new stable version.
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} on your {OCP} cluster.
```

Comment:
@varshab1210 I _think_ since this is the 1.13 version of the docs that we don't need to mention the version in the documentation text, but perhaps @Dhruv-Soni11 has some insight here on what the best practice is here.

Reply:
Makes sense, thanks Jonathan.

Reply:
Yes, it is implied that the user understands that is the GitOps 1.13 document as changes to version numbers will be made automatically in the published documentation.